package.json

```diff
@@ -26,8 +27,8 @@
     "bower": "latest",
     "gulp": "^3.9.0",
     "main-bower-files": "^2.9.0",
-    "nconf": "^0.7.2",
-    "money-to-prisoners-gulp-tasks": "ministryofjustice/money-to-prisoners-gulp-tasks#0.1.0"
+    "money-to-prisoners-gulp-tasks": "ministryofjustice/money-to-prisoners-gulp-tasks#0.2.0",
```

Comment:
I think money-to-prisoners-gulp-tasks should currently be at 0.0.2 since the last change was a bug fix, not a new feature.